# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-vesa-kern-cont.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.11/drivers/video/fbdev/vesafb.c.vanilla	2024-10-08 23:22:56.439977841 +0200
+++ linux-6.11/drivers/video/fbdev/vesafb.c	2024-10-08 23:25:25.279974589 +0200
@@ -356,8 +356,8 @@
 		if (pmi_base[3]) {
 			printk(KERN_INFO "vesafb: pmi: ports = ");
 			for (i = pmi_base[3]/2; pmi_base[i] != 0xffff; i++)
-				printk("%x ", pmi_base[i]);
-			printk("\n");
+				printk(KERN_CONT "%x ", pmi_base[i]);
+			printk(KERN_CONT "\n");
 			if (pmi_base[i] != 0xffff) {
 				/*
 				 * memory areas not supported (yet?)
